### PR TITLE
Created Procedure for Setting Up a Scaler 

### PR DESCRIPTION
**Pull request by shubhusion**
### Description of Changes

Created a procedure for setting up a scaler in KEDA.

### Checklist
- [x] Commits are signed with Developer Certificate of Origin (DCO)

### Issue Fixed
Fixes [kedacore/keda-docs#1368](https://github.com/kedacore/keda-docs/issues/1368)

### CODE REVIEW

**Comment by shubhusion** [comment]
@tomkerkhove please review this issue

**Comment by shubhusion** [comment]
@zroubalik can you please review

File: content/docs/2.14/operate/setupscaler.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+   - Follow the [official kubectl installation guide](https://kubernetes.io/docs/tasks/tools/#kubectl) to install `kubectl` on your operating system.
+   - Once installed, configure `kubectl` to communicate with your Kubernetes cluster by following the cluster-specific instructions provided by your Kubernetes setup.
+
+## Step 1: Identify the Scaler You Need
+
+KEDA supports various scalers that correspond to different event sources or triggers. Determining the right scaler is crucial for scaling your application based on the desired event source.
+
+1. Visit the [KEDA Scalers documentation](https://keda.sh/docs/2.14/scalers/) and browse through the list of available scalers.
+2. Identify the scaler that matches the event source you want to use for scaling your application. For example:
+   - If you want to scale based on incoming HTTP traffic, you would need the **HTTP scaler**.
```
Comment (JorTurFer) [inline content/docs/2.14/operate/setupscaler.md:29]: We don't have any HTTP Scaler atm, the closest one is the HTTP Add-on which is in beta status atm. I'm not 100% if we should include here (at least without a note).
@tomkerkhove  @zroubalik ?

Comment (shubhusion) [reply content/docs/2.14/operate/setupscaler.md:29]: @JorTurFer so How should i proceed ? remove it or add a note

Comment (zroubalik) [reply content/docs/2.14/operate/setupscaler.md:29]: @shubhusion add a note please

File: content/docs/2.14/operate/setupscaler.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
++++
+title = "Setupscaler"
+weight = 500
+description = "Procedure to Setup a Scaler in KEDA"
```
Comment (zroubalik) [inline content/docs/2.14/operate/setupscaler.md:4]:
```suggestion
description = "Procedure to Setup a Autoscaling with KEDA"
```

File: content/docs/2.15/operate/setupscaler.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
++++
+title = "Setupscaler"
+weight = 500
+description = "Procedure to Setup a Scaler in KEDA"
```
Comment (zroubalik) [inline content/docs/2.15/operate/setupscaler.md:4]: description = "Procedure to Setup a Autoscaling with KEDA"

File: content/docs/2.14/operate/setupscaler.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
++++
+title = "Setupscaler"
```
Comment (zroubalik) [inline content/docs/2.14/operate/setupscaler.md:2]:
```suggestion
title = "Setup Autoscaling with KEDA"
```

File: content/docs/2.15/operate/setupscaler.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
++++
+title = "Setupscaler"
```
Comment (zroubalik) [inline content/docs/2.15/operate/setupscaler.md:2]:
```suggestion
title = "Setup Autoscaling with KEDA"
```

File: content/docs/2.15/operate/setupscaler.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+   - Follow the [official kubectl installation guide](https://kubernetes.io/docs/tasks/tools/#kubectl) to install `kubectl` on your operating system.
+   - Once installed, configure `kubectl` to communicate with your Kubernetes cluster by following the cluster-specific instructions provided by your Kubernetes setup.
+
+## Step 1: Identify the Scaler You Need
+
+KEDA supports various scalers that correspond to different event sources or triggers. Determining the right scaler is crucial for scaling your application based on the desired event source.
+
+1. Visit the [KEDA Scalers documentation](https://keda.sh/docs/2.14/scalers/) and browse through the list of available scalers.
+2. Identify the scaler that matches the event source you want to use for scaling your application. For example:
+   - If you want to scale based on incoming HTTP traffic, you would need the **HTTP Add-on**.
```
Comment (zroubalik) [inline content/docs/2.15/operate/setupscaler.md:29]:
```suggestion
   - If you want to scale based on incoming HTTP traffic, you would need the [HTTP Add-on](https://kedacore.github.io/http-add-on/).
```

File: content/docs/2.14/operate/setupscaler.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+   - Follow the [official kubectl installation guide](https://kubernetes.io/docs/tasks/tools/#kubectl) to install `kubectl` on your operating system.
+   - Once installed, configure `kubectl` to communicate with your Kubernetes cluster by following the cluster-specific instructions provided by your Kubernetes setup.
+
+## Step 1: Identify the Scaler You Need
+
+KEDA supports various scalers that correspond to different event sources or triggers. Determining the right scaler is crucial for scaling your application based on the desired event source.
+
+1. Visit the [KEDA Scalers documentation](https://keda.sh/docs/2.14/scalers/) and browse through the list of available scalers.
+2. Identify the scaler that matches the event source you want to use for scaling your application. For example:
+   - If you want to scale based on incoming HTTP traffic, you would need the **HTTP Add-on**.
```
Comment (zroubalik) [inline content/docs/2.14/operate/setupscaler.md:29]:
```suggestion
   - If you want to scale based on incoming HTTP traffic, you would need the [HTTP Add-on](https://kedacore.github.io/http-add-on/).
```

**Review by zroubalik — COMMENTED** [review]
I think we should move this into Getting Started section: https://keda.sh/docs/2.15/

At least for 2.15

**Comment by shubhusion** [comment]
> I think we should move this into Getting Started section: https://keda.sh/docs/2.15/
> 
> At least for 2.15

Thanks for the review. I will make the necessary changes as specified.

**Comment by shubhusion** [comment]
I guess I made a mistake while rebasing. I will open a new PR with the required commits

**Pull request closed without merging**